packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -59,13 +59,19 @@ export const styles = (theme) => ({
         ),
       },
     },
+    '&$disabled': {
+      color: theme.palette.action.disabled,
+      opacity: theme.palette.action.disabledOpacity,
+    },
   },
   /* Pseudo-class applied to the content element when expanded. */
   expanded: {},
   /* Pseudo-class applied to the content element when selected. */
   selected: {},
   /* Pseudo-class applied to the content element when focused. */
   focused: {},
+  /* Styles applied to the element when disabled */
```

Comment:
```suggestion
  /* Pseudo-class applied to the element when disabled. */
```

Reply:
Thanks, updated it as part of the rebase but I think it got overwritten at some point.